include/aws/common/json.h

```diff
@@ -24,7 +24,7 @@ struct aws_json_value;
  * @return A new string aws_json_value
  */
 AWS_COMMON_API
-struct aws_json_value *aws_json_new_string(struct aws_allocator *allocator, struct aws_byte_cursor string);
+struct aws_json_value *aws_json_string_new(struct aws_allocator *allocator, struct aws_byte_cursor string);
```

Comment:
My suggstion was intended to be `aws_json_value_new_*` ie `aws_json_value_new_string` `aws_json_value_new_number` etc...